cmds/wifi/wifi.go

```diff
@@ -29,20 +32,32 @@ func init() {
 }
 
 func main() {
+	var (
+		iface = flag.String("i", "wlan0", "interface to use")
```

Comment:
well, the flag can be global :) i was just talking about the other three. but either way, it's a small program, so whatever.

Reply:
yeah, it can be made global should it ever matter, which at the moment it does not :-)